shared/notifications/index.js

```diff
@@ -0,0 +1,67 @@
+export const UI_NOTIFICATIONS = {
+  1: {
+    id: 1,
+    title: 'notifications#1Title',
+    description: 'notifications#1Description',
+    date: '02/22/2020',
```

Comment:
:scream: Why this date format?

We use milliseconds-since-epoch in most places to store dates. If we wanted something that is _just_ a date and not a datetime we could use ISO-8601 (e.g. '2020-02-22'), but I don't think there's any place for non-standard formats like this.

Reply:
This was addressed within 45aa92e7a